puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/updateJobs.groovy

```diff
@@ -0,0 +1,39 @@
+pipeline {
+    agent { label 'foreman' }
+
+    stages {
+        stage('Update Jenkins Jobs') {
```

Comment:
Some of my comments got lost in the restructure but I think splitting this into two stages would be nice.